app/assets/stylesheets/_search_result.scss

```diff
@@ -38,22 +39,28 @@
           margin: 10px auto;
           img { margin: 0; }
         }
-      }
+       }
+     }
+    .button-action {
+      float: left;
+      padding-top: 10px;
     }
-    .examples-container {
-      .translation-gloss {
-        @include montserrat-regular;
-        font-size: $h6-font-size;
-        margin: 0 0 10px 0;
-      }
+    .add-to-vocab-btn { padding-right: 45px; }
+  }
+
+  .examples-container {
```

Comment:
is there a reason this block was pulled a level up from being nested in the search-result--container class to just being nested in the search-result classe? Also can the indentation of all of this be tidied up please.

Reply:
@Dan Code Climate did not like the nesting. Only did not complain when I pulled it out of the search results class.

Reply:
Ok it looks like its only used in one place right now anyway, and it is sort of its own element in that its a more specific search result container so that seems fine as long as it hasnt had any affect on how it displays. I'll resolve this comment once the indentation/brackets have been tidied up.

Reply:
@Rabid-Dan Do you mean the one liner when you refer to the indentation?